### PR TITLE
Fix ccd-definition-processor and yarn upgrade :)

### PR DESCRIPTION
**Pull request by furnivall**

### CODE REVIEW

File: docker/importer.Dockerfile [hunk 1]
```diff
@@ -12,5 +13,6 @@ RUN apk add --no-cache curl jq zip unzip git
 COPY --from=base . .
 COPY ./data /data
 COPY ./data/ccd-template.xlsx /opt/ccd-definition-processor/data
-
+USER root
+RUN chmod -R 777 /opt/ccd-definition-processor
```
Comment (timja) [inline docker/importer.Dockerfile:17]: is this needed?

File: Dockerfile [hunk 2]
```diff
@@ -1,20 +1,23 @@
 # ---- Base image - order important ----
-FROM hmcts/ccd-definition-processor:latest as base
-
+FROM hmctspublic.azurecr.io/ccd/definition-processor:latest as base
+USER root
+RUN usermod -u 1000 hmcts && \
+    groupmod -g 1000 hmcts
```
[the remark this answers is not in the document]
Comment (furnivall) [reply Dockerfile:5]: This is needed.

File: docker/importer.Dockerfile (hunk 1 above)
Comment (furnivall) [reply docker/importer.Dockerfile:17]: This I'm not so sure about.

File: Dockerfile [hunk 3]
```diff
@@ -1,19 +1,23 @@
 # ---- Base image - order important ----
-FROM hmctspublic.azurecr.io/ccd/definition-processor:dbyccu as base
+FROM hmctspublic.azurecr.io/ccd/definition-processor:latest as base
+USER root
+RUN usermod -u 1000 hmcts && \
```
Comment (timja) [inline Dockerfile:4]: shouldn't we change this back to 1000 here then?
https://github.com/hmcts/ccd-definition-processor/blob/a217ea91595b403b90ad7fd88f8c9ff9764dd059/Dockerfile#L9

**Review by jarekPierchala — APPROVED** [review]

**Review by AndreiTrushkov — APPROVED** [review]

**Pull request merged**